index.js

```diff
@@ -106,35 +106,41 @@ exports.generate = async (inputFile, outputDirectory, isApiMonolith, userProvide
 				TYPES_DIRECTORY: isApiMonolith ? "../apiModelTypes" : "./apiModelTypes",
 				BASE_PATH: serviceFileBasePath,
 				FUNCTIONS: _.map(paths, (pathConfig) => {
+					const pathParams = _.filter(pathConfig.parameters, (parameter) => {
+						return _.toLower(parameter.in) === "path";
+					});
```

Comment:
For some reason I could get incorporate the use of _,filter() directly on line 122 or conditionally returning only queries with parameter.in === "path". So I created this variable to hold them.